decidim-admin/app/controllers/concerns/decidim/admin/content_blocks/landing_page.rb

```diff
@@ -29,6 +34,40 @@ def update
             end
           end
 
+          # Method that specifies the i18n scope for this controller. This can be overwritten in your controller.
+          # so that you can pass a different scope to the i18n methods.
+          def i18n_scope = "decidim.admin.organization_homepage"
```

Comment:
Referring to the `"decidim.admin.organization_homepage"` scope in `landing_page` doesn't sounds right. How about using `"decidim.admin.content_blocks"` directly?